lambda/templates/rotate_keycloak_secrets_policy.json.tpl

```diff
@@ -0,0 +1,41 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "logs:CreateLogStream",
```

Comment:
Could you put these in alphabetical order?

Reply:
I've put these in order and the resources as well.